task_templates/README.md

```diff
@@ -1,50 +1,70 @@
+  classification problems
+* `python_multiclass_classification` - Implements a linear classifier with SGD training
+* `python_regression` - Implements a linear regressor with SGD training
+* `python_regression_with_custom_class` - Uses a custom python class, CustomCalibrator, so that we
+  can store the complex state inside the object and then re-use it during scoring.
+* `r_classification` - Implements a GLM binary classifier in R
+* `r_regression` - Implements a GLM regressor in R
 
 ## Pipelines
-Except for where noted, pipelines are also verified within our functional test framework. 
-
- * `python3_anomaly_detection`
- * `python3_pytorch`
- * `python3_sklearn_binary_schema_validation`
- * `python3_calibrated_anomaly_detection`
- * `python3_pytorch_multiclass`
- * `python3_sklearn_multiclass`
- * `python3_sklearn_with_custom_classes` - This pipeline shows you how to not use one of the main libraries (untested)
- * `python3_keras_joblib`
- * `python3_sklearn_binary`
- * `python3_sklearn_regression`
- * `python3_sparse`
- * `python3_keras_vizai_joblib`
- * `python3_sklearn_binary_hyperparameters`
- * `python3_xgboost`
- * `r_lang`
- * `r_lang_hyperparameters`
- * `simple`
+
+Except for where noted, pipelines are also verified within our functional test framework.
+
+* `python3_anomaly_detection` - Provides an uncalibrated Sklearn anomaly pipeline
+* `python3_pytorch` - Provides a pytorch pipeline. Also uses the transform hook
+* `python3_sklearn_binary_schema_validation` - Shows how to supply a model-metadata.yaml file to
+  dictate which data types are allowed in this bp
+* `python3_calibrated_anomaly_detection` - Provides a calibrated Sklearn anomaly pipeline
+* `python3_pytorch_multiclass` - Parses the passed in class_labels file. Also uses transform, and
+  also relies on the internal PyTorch prediction implementation
+* `python3_sklearn_multiclass` - Handles text, numeric and categorical inputs. Relies on sklearn
+  dropin env and predictor.
+* `python3_sklearn_with_custom_classes` - This pipeline shows you how to build an estimator
+  independent from any library [Not Verified]
+* `python3_keras_joblib` - Contains the option for both binary classification and regression.
+  Serializes to the h5 format
+* `python3_sklearn_binary` - Preprocessing with numeric, categorical and text, then SVD, with a
+  linear model estimator at the end
+* `python3_sklearn_regression` - Preprocessing with numeric, categorical and text, then SVD, with a
+  Ridge regression estimator at the end
+* `python3_sparse` - A pipeline that intakes sparse data (csr_matrix) from an MTX file
+* `python3_keras_vizai_joblib` - Trains a keras model on base64 encoded images.
+* `python3_xgboost` - XGBoost also has a DRUM predictor provided that knows how to predict all
+  XGBoost models
+* `r_lang` - This R model can support either binary classification or regression out of the box.
```

Comment:
since we have examples of R estimators, let's describe it as a pipeline. E.g. "Shows an example of a pipeline defined in R"